### PR TITLE
fix: remove pagination from dashboard microblock transactions panel

### LINKED ISSUE #248: When using a mobile device when selecting a microblock with one transaction it looks like it has three instead.
### Context information

application version: pr-244
device: Samsung Galaxy s22
browser: All browsers
operating system: Android

### Steps to reproduce
1.Go to the dashboard
2.Scroll down to the microblocks
3.Select on microblock with 4 transactions, then 3, 2 and finally one
4.Observe the amount of transactions that can be scrolled bellow them
### What is expected?
The amount of the transactions appearing bellow should much the number written on the box
### What is actually happening?
When selecting o vox with three transactions or more only three dots can be seen. 
Then when we select a box with two the dots and scrollable transactions decrease to two.
But then when we select a box with one three dots appears again. The same transaction is repeated three times.

https://github.com/aeternity/aescan/assets/42606212/38afdfcb-e8d6-4ee4-bace-11788d93694e

### LINKED ISSUE #354: avoid screen jumping for tx-pagination in microblock section on dashboard
### Please describe the problem that your request should resolve.
when clicking on the pagination in tx-block in microblock section, the screen jumps unexpectetly

### Describe the solution you'd like
don't jump around 😅 

### Is it already possible to achieve the same outcome in any other way? If so, how?
nope

### Additional context
n/a

### PR DESCRIPTION
**Pull request by lukeromanowicz**
## Description
Resolves #248 
Resolves #354

Removes pagination from dashboard microblock transactions panel

## Demo

https://github.com/aeternity/aescan/assets/46789227/462541c4-ea31-48de-ad81-439c21d37613

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] My change does not require a change to the documentation.

### CODE REVIEW

File: src/components/DashboardMicroblockTransactionsTable.vue [hunk 1]
```diff
@@ -1,74 +1,49 @@
...
-const props = defineProps({
+defineProps({
   transactions: {
     required: true,
     type: Object,
   },
 })
-
-function loadPrevTransactions() {
-  fetchSelectedMicroblockTransactions(props.transactions.prev)
```
Comment (janmichek) [inline src/components/DashboardMicroblockTransactionsTable.vue:60]: I guess the param is not needed anymore in the function definition

**Review by janmichek — CHANGES_REQUESTED** [review]
Good job, I like this simplification. 
One more suggested adjustment.

**Review by CharisSiarampalis — APPROVED** [review]

File: src/components/DashboardMicroblockTransactionsTable.vue (hunk 1 above)
Comment (janmichek) [reply src/components/DashboardMicroblockTransactionsTable.vue:60]: query params does not seems to be used anywhere in the app
image

Comment (lukeromanowicz) [reply src/components/DashboardMicroblockTransactionsTable.vue:60]: Fixed in the latest commit

**Review by janmichek — APPROVED** [review]
Perfecto 👌

**Review by michele-franchi — COMMENTED** [review]
In the desktop view pagination could be useful, why did you decide to also remove it? I don't see the same issue mobile has.

**Review by michele-franchi — APPROVED** [review]
I don't have a strong opinion about this solution but I also think it could work.

**Pull request merged**